src/platform/monitoring/DowntimeNotification/tests/DowntimeNotification.cypress.spec.js

```diff
@@ -0,0 +1,68 @@
+import moment from 'moment';
+import Timeouts from 'platform/testing/e2e/timeouts';
+import FacilityHelpers from 'applications/facility-locator/tests/facility-helpers-cypress';
+
+Cypress.Commands.add('createDowntimeNotificationIntercept', data => {
+  cy.intercept('GET', '/v0/maintenance_windows', { data });
+});
+
+const beforeNow = moment()
+  .subtract(1, 'minute')
+  .toISOString();
+const withinHour = moment()
+  .add(1, 'hour')
+  .subtract(1, 'minute')
+  .toISOString();
+const endTime = moment()
+  .add(6, 'hour')
+  .toISOString();
+
+const selectors = {
+  app: '.facility-locator',
+  statusDown: '.usa-alert-heading',
+  statusDownApproachingModal:
+    '[data-status="downtimeApproaching"] #downtime-approaching-modal',
+};
+
+describe('Downtime Notification Test', () => {
+  it('Shows the correct notification in all scenarios', () => {
```

Comment:
I'd like to see this as separate expectations, but this is okay

Reply:
![image](https://user-images.githubusercontent.com/34250637/127513325-e0fbca16-905e-47f1-a0c0-8ffbdd641520.png)

Done!